usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_transaction.md

```diff
@@ -0,0 +1,82 @@
+## [Spending by Transaction](#spending-by-transaction)
+**Route:** `/api/v2/search/spending_by_transaction/`
+
+**Method:** `POST`
+
+This route takes keyword search fields, and returns the fields of the searched term.
```

Comment:
This route takes keyword search terms and returns awards where a certain subset of fields match against the term. For more information about which fields are searched, refer to https://usaspending-help.zendesk.com/hc/en-us/articles/360001255774-Keyword-Search-Question-How-does-the-Keyword-Search-work-

Reply:
Done